backend/src/models/weekly-action-items.ts

```diff
@@ -0,0 +1,7 @@
+import mongoose from 'mongoose';
+import { IWeeklyActionItems, WeeklyActionItemsMongooseSchema } from 'shared-types';
+
+const WeeklyActionItemsSchema = new mongoose.Schema<IWeeklyActionItems>(WeeklyActionItemsMongooseSchema.schema);
+
+WeeklyActionItemsSchema.index({ businessId: 1, date: 1 });
```

Comment:
What is this? Index it in the WeeklyActionItemsMongooseSchema. You can add index: true to the required fields to be indexed